src/clr-angular/utils/focus-trap/focus-trap.directive.spec.ts

```diff
@@ -12,50 +12,101 @@ import { NoopAnimationsModule } from '@angular/platform-browser/animations';
 import { itIgnore } from '../../../../tests/tests.helpers';
 import { ClrModal } from '../../modal/modal';
 import { ClrModalModule } from '../../modal/modal.module';
-
 import { FocusTrapDirective } from './focus-trap.directive';
 import { ClrFocusTrapModule } from './focus-trap.module';
 
 describe('FocusTrap', () => {
   let fixture: ComponentFixture<any>;
   let compiled: any;
   let component: TestComponent;
-  let directive: FocusTrapDirective;
+
   let lastInput: HTMLElement;
-  const tabEvent = { shiftKey: false, keyCode: 9, preventDefault: () => {} };
+
+  let directiveDebugElement: DebugElement;
+  let directiveInstance: FocusTrapDirective;
+  let directiveElement: HTMLElement;
 
   describe('default behavior', () => {
     beforeEach(() => {
       TestBed.configureTestingModule({ imports: [ClrFocusTrapModule], declarations: [TestComponent] });
-
       fixture = TestBed.createComponent(TestComponent);
-      component = fixture.componentInstance;
       fixture.detectChanges();
-      compiled = fixture.nativeElement;
-      directive = fixture.debugElement.query(By.directive(FocusTrapDirective)).injector.get(FocusTrapDirective);
 
+      component = fixture.componentInstance;
+      compiled = fixture.nativeElement;
       lastInput = compiled.querySelector('#last');
+
+      directiveDebugElement = fixture.debugElement.query(By.directive(FocusTrapDirective));
+      directiveElement = directiveDebugElement.nativeElement;
+      directiveInstance = directiveDebugElement.injector.get(FocusTrapDirective);
     });
 
     afterEach(() => {
       fixture.destroy();
     });
 
     it('should create directive', () => {
-      expect(directive).toBeTruthy();
+      expect(directiveInstance).toBeTruthy();
     });
 
     it('should add tabindex attribute with value zero', () => {
-      directive.ngAfterViewInit();
-      const element: HTMLElement = directive.elementRef.nativeElement;
-      expect(element.getAttribute('tabindex')).toEqual('0');
```

Comment:
Thanks for cleaning that one up, I don't remember why it was manually doing that.